resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+openapi: 3.0.3
+info:
+  title: Lifetime ISA
+  description: |
+    # Save reference numbers
+    
+    When you report or get information from the API, you will get a unique reference number. You should save all reference numbers because you may need them for other API calls.
+    
+    # Testing
+    
+    You can use the sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing).
+
+    It does not currently support [stateful behaviour](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing/stateful-behaviour), but you can use the payloads described in **Test data** sections of each endpoint to test specific scenarios.
+
+    You must set up a test user which is an organisation for this API using the [Create Test User API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user/1.0#_create-a-test-user-which-is-an-organisation_post_accordion).
```

Comment:
Can you remove this? This has been duplicated is already in the common folder

Reply:
OK